helm/kong-app/templates/crd-install/crd-job.yaml

```diff
@@ -21,7 +21,9 @@ spec:
     spec:
       serviceAccountName: {{ template "kong.name.crdInstall" . }}
       securityContext:
-      {{- include "kong.podsecuritycontext" . | nindent 8 }}
```

Comment:
or we could just set defaults for `kong.podsecuritycontext` in the values.yaml?

Reply:
The container image is hardcoded and needs exactly this securityContext. We could make the container image configurable as well, then it would make sense to have this context configurable.

Reply:
That is a good point. Making that configurable could help to quick fix in case the image needs to be changed for a newer version. Instead of making everywhere new releases crd-install is used. But maybe also overthinking here right now? Feel free to choose to just merge..

Reply:
Yeah this whole construct should be a subchart as previously discussed on many places